loop_live.py

```
@@ -0,0 +1,644 @@
+            ProjectSetupTool()
+        )
+        ic(tool_collection)
+        display.add_message("system", tool_collection.get_tool_names_as_string())
+        system = BetaTextBlockParam(type="text", text=SYSTEM_PROMPT_FILE)
+        output_manager = OutputManager(display)
+        client = Anthropic(api_key=api_key)
+        i = 0
+        running = True
+        token_tracker = TokenTracker(display)
+        enable_prompt_caching = True
+        betas = [COMPUTER_USE_BETA_FLAG, PROMPT_CACHING_BETA_FLAG]
```

Ruff (0.8.2)

`COMPUTER_USE_BETA_FLAG` may be undefined, or defined from star imports

(F405)